test/e2e/Tests/Tests/OrchestrationQueryTests.cs

```diff
@@ -23,6 +23,7 @@ public OrchestrationQueryTests(FunctionAppFixture fixture, ITestOutputHelper tes
 
 
     [Fact]
+    [Trait("DTS", "Skip")] // Skip this test for now as there is a bug in DTS with GetAllInstances
```

Comment:
Can you provide a GitHub or ADO link? We should track these bugs and make sure they get fixed.